nats/js/client.py

```diff
@@ -362,22 +357,54 @@ async def main():
         except nats.js.errors.NotFoundError:
             # If not found then attempt to create with the defaults.
             if config is None:
-                config = api.ConsumerConfig(
-                    ack_policy=api.AckPolicy.EXPLICIT,
-                )
+                config = api.ConsumerConfig()
             # Auto created consumers use the filter subject.
             config.filter_subject = subject
             config.durable_name = durable
             await self._jsm.add_consumer(stream, config=config)
 
-        # FIXME: Make this inbox prefix customizable.
-        deliver = api.InboxPrefix[:]
-        deliver.extend(self._nc._nuid.next())
+        return await self.pull_subscribe_fast(durable=durable, stream=stream)
+
+    async def pull_subscribe_fast(
```

Comment:
There is a somewhat similar option in the nats.go client called `nats.Bind`: https://github.com/nats-io/nats.go/blob/main/js.go#L2195-L2197
How about renaming to `pull_subscribe_bind` to keep the semantics more or less the same?

Reply:
Great idea! Fixed.